### ISSUE TITLE
Army Reveal in network matches (Random Balance)

### ISSUE THREAD

**Issue by ImTimK** [TheSuperHackers/GeneralsGamePatch#1508]
The patch should reveal armies in network games aswell, or atleast needs to be an option.

**Comment by xezon** [comment]
Ideally is a Room Setting to reveal armies and colors in load screen. This requires Thyme however. 

Base game can only Reveal or Hide always. As long as that is the case, I prefer to keep it as per Original setup. This has the least ramifications.

**Comment by MTKing4** [comment]
> Base game can only Reveal or Hide always. As long as that is the case, I prefer to keep it as per Original setup. This has the least ramifications.

I disagree, i think it should be reveal always for the time being until you make that an option, it makes no sense why you would want to hide it, it helps with balance, many mods have it already and it feels good, i don't think anyone would hate to see it from the competitive scene, in fact it might even be a selling point

**Comment by xezon** [comment]
If we do this we will annihilate all players who enjoy a true Random match with hidden factions. If we do this, then this should be a community vote.

**Comment by MTKing4** [comment]
I doubt that there is an audience for that, but sure we can collect votes on that,  besides it's just temporary until thyme fix arrives

**Comment by ImTimK** [comment]
Having options is always good, I really like the idea of having a toggle button in the lobby same like limiting SW's.

**Comment by MTKing4** [comment]
I do too, but if we had no choice, we should go with revealed armies not the opposite

**Comment by HeissesFeuer** [comment]
without thyme, I think the reveal armies should stay like it is now. having all armies show on loading screen would probably ruin most free for all games. the players with the stronger armies will have a higher chance of getting teamed in the first minutes of the game.
besides it's not really a problem for 1v1s since you can type what armies you have or have an in and out to pick armies

**Comment by xezon** [comment]
I tested times.

* Overcharge Suicide Time: 33 seconds
* Overcharge Suicide + Repair Time: 99 seconds

Max 33 seconds for overcharge is not bad. 99 seconds for overcharge is pretty good. This does not look in need of touching.

Close?

**Comment by MTKing4** [comment]
> the players with the stronger armies will have a higher chance of getting teamed in the first minutes of the game.

With a Balance Patch, there won't be any _Stronger_ armies, so this point doesn't matter

> it's not really a problem for 1v1s since you can type what armies you have or have an in and out to pick armies

Why would anyone prefer typing armies and doing in and outs over random balance, which saves a lot of time in challenges and removes that inconvenience at the start of the game.

**Comment by HeissesFeuer** [comment]
I don't totally agree, does revealing spots come with revealing the armies?
if yes, coming from a free for all perspective, you need to usually scan or send an rpg forward to know who is next to you.
if you get to know who is your neighbour from the very beginning your strategy would change.
Let's say you are a strong player, and the spots are revealed from the beginning, you are more likely to get doubled when the game starts.
If you can reveal armies without revealing spots I would be more likely to agree with you.

**Comment by xezon** [comment]
Random Army, Random Spot and Random Color can be changed independently.

**Comment by ImTimK** [comment]
"secret" armies and spots definitely has a place in this game and should remain optional for sure.
